frontend/pages/_app.tsx

```diff
@@ -48,6 +48,7 @@ export default function App({
   const { locale, defaultLocale, pathname } = router;
   const messages = locales[locale];
   return (
+    // eslint-disable-next-line @typescript-eslint/no-unused-vars
```

Comment:
I tried looking into this more, and I can't seem to replicate this at all... Maybe we can pair on this to figure out why it's happening?